src/_macosx.m

```diff
@@ -26,13 +23,6 @@
 #define COMPILING_FOR_10_10
 #endif
 
-/* Use Atsui for Mac OS X 10.4, CoreText for Mac OS X 10.5 */
-#ifndef COMPILING_FOR_10_5
```

Comment:
I had actually done a lot of cleanup and warning fixes to my codebase and accidentally included this. This just removes an unused ngc variable (I think it may have been used before @mdboom's conversion to agg) and the carbon include (Cocoa is already included irregardless of the OSX version earlier).